NUModules/NUModule.j

```diff
@@ -3339,7 +3417,16 @@ NUModuleTabViewModeIcon                = 2;
     _categories = someCategories;
     [self didChangeValueForKey:@"categories"];
 
-    _usesPagination = ![_categories count];
+    // Use pagination only if all categories specify a context
+    _usesPagination = YES;
+    for (var i = 0; i < [_categories count]; i++)
+    {
+        if (![_categories[i] contextIdentifier])
+        {
+            _usesPagination = NO;
+            break;
+        }
+    }
```

Comment:
I found 56 results, which is not a big deal. It will be part of the PR we will suggest on our repository